src/ragger/backend/speculos.py

```diff
@@ -160,7 +169,7 @@ def left_click(self) -> None:
     def both_click(self) -> None:
         self._client.press_and_release("both")
 
-    def finger_touch(self, x: int = 0, y: int = 0, delay: float = 0.5) -> None:
+    def finger_touch(self, x: int = 0, y: int = 0, delay: float = 0.1) -> None:
```

Comment:
```suggestion
    def finger_touch(self, x: int = 0, y: int = 0, delay: float = TICKER_DELAY) -> None:
```
(not sure it makes total sense though)

Reply:
I prefer keeping the explicit 0.1 here.